pkgpanda/__init__.py

```diff
@@ -356,7 +356,7 @@ def load(self, id: str):
         filename = os.path.join(path, "pkginfo.json")
         try:
             pkginfo = load_json(filename)
-        except FileNotFoundError as ex:
+        except OSError as ex:
```

Comment:
The path can exist but not be a directory. https://docs.python.org/3/library/exceptions.html#NotADirectoryError
> Raised when a directory operation (such as `os.listdir()`) is requested on something which is not a directory. Corresponds to errno `ENOTDIR`.